wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java

```diff
@@ -377,7 +378,21 @@ public Wiki createWiki(String wikiType, String owner) throws WikiException {
     wikiPreferences.setWikiPreferencesSyntax(wikiPreferencesSyntax);
     wiki.setPreferences(wikiPreferences);
     Wiki createdWiki = dataStorage.createWiki(wiki);
-
+    StringBuilder sb = new StringBuilder("= Welcome to ");
+    String spaceName = owner;

```

Comment:
spaceName is not an accurate name, since it is not always a space, it can be misleading. I would prefer wikiLabel for example.